src/ngraph/op/convolution.cpp

```diff
@@ -443,16 +473,16 @@ void op::v1::ConvolutionBackpropFilters::validate_and_infer_types()
                           delta_shape,
                           ").");
 
-    set_output_type(0, forward_result_et, m_filters_shape);
+    set_output_type(0, forward_result_et, get_filters_shape());
```

Comment:
if get_filters_shape() returns empty shape then set_output_type to PartialShape::Dynamic(). Also add set_input_relevant_to_shape